governance/votes/Trial Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,191 @@
+|Harmony|[0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2)|1.92|
+|Vote Wizard|[0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9e72629df4fcaa2c2f5813fbbdc55064345431b1)|1.6|
+|Navigator|[0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072)|0.32|
+
+
+
+### Ecosystem Actor DAI Budget Stream
+
+As per [MIP106 Article 7.4.2.1A](https://mips.makerdao.com/mips/details/MIP106#7-4-2-1a-), and ratified in [Amendment Subproposal MIP102c2-SP8](https://vote.makerdao.com/polling/QmdnSKPu#poll-detail), the following DAI budget streams will be created if this executive proposal passes:
+
+| Budget | DAI Amount | Start Date | End Date | Destination Address |
+|--|--|--|--|------|
+|Chronicle Labs Auditor Wallet|3,721,800|2023-07-01|2024-06-30|[0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f](https://etherscan.io/address/0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f#readContract)|
+|Jet Stream Auditor Wallet|2,964,006|2023-07-01|2024-12-31|[0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962](https://etherscan.io/address/0xf478a08c41ad06e8d957d5e6b6bcde7452cee962)|
+
+### Ecosystem Actor MKR Budget Stream
+
+As per [MIP106 Article 7.4.2.1A](https://mips.makerdao.com/mips/details/MIP106#7-4-2-1a-), and ratified in [Amendment Subproposal MIP102c2-SP8](https://vote.makerdao.com/polling/QmdnSKPu#poll-detail), the following MKR budget streams will be created if this executive proposal passes:
+
+| Budget | MKR Amount | Start Date | End Date | Destination Address |
+|--|--|--|--|------|
+|Chronicle Labs Auditor Wallet|2,216.4|2023-07-01|2024-06-30|[0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f](https://etherscan.io/address/0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f#readContract)|
+|Jet Stream Auditor Wallet|1,619.93|2023-06-26| 2024-12-31|[0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962](https://etherscan.io/address/0xf478a08c41ad06e8d957d5e6b6bcde7452cee962)|
+
+### Ecosystem Actor DAI Transfer
+- **494,001 DAI** will be transferred to [Jet Stream Ecosystem Actor](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-jetstream/21054) at [0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962](https://etherscan.io/address/0xf478a08c41ad06e8d957d5e6b6bcde7452cee962) as per this [poll](https://vote.makerdao.com/polling/QmdnSKPu#poll-detail) and [MIP](https://mips.makerdao.com/mips/details/MIP108#9-launch-project)
+
+### Core Unit Vesting Transfers 
+As per previously ratified governance proposals, the following vesting transfers will take place if this executive proposal passes.
+
+- **125 MKR** will be transferred to the [Deco Fixed Rate Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP23) at [0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7](https://etherscan.io/address/0xf482d1031e5b172d42b2daa1b6e5cbf6519596f7) as per this [budget MIP](https://mips.makerdao.com/mips/details/MIP40c3SP36#sentence-summary).
```

Comment:
Calling attention to correct consistency in tense here. '125 MKR _will be transferred_' rather than 'Transfer 125 MKR to...'

Reply:
In comparison to stability fee stuff